src/main/java/ua/tc/marketplace/util/mapper/ClassificationAttributeMapper.java

```diff
@@ -0,0 +1,34 @@
+package ua.tc.marketplace.util.mapper;
+
+import org.springframework.stereotype.Component;
+import ua.tc.marketplace.exception.model.MappingException;
+import ua.tc.marketplace.model.dto.ClassificationAttributeDto;
+import ua.tc.marketplace.model.entity.ClassificationAttribute;
+
+@Component
+public class ClassificationAttributeMapper {
```

Comment:
delete this whole calss. Learn about Mapstruct mappers and how to use them. Also see DemoMapper interface. Ask questions in our chat